src/main.rs

```diff
@@ -145,6 +145,13 @@ fn inner_main(language: &mut Option<Language>) -> errors::Result<()> {
       renderer.window().set_outer_position(PhysicalPosition::new(window.x, window.y));
    }
 
+   // Maximize the window, if it was maximized last time.
+   // NOTE: winit is a bit buggy and WindowBuilder::with_maximized does not
+   // make window maximized, but Window::set_maximized does.
+   if let Some(window) = &config().window {
+      renderer.window().set_maximized(window.maximized);
+   }
+
```

Comment:
I think this `if let` can be merged with the one above (not visible in the diff so I unfortunately can't suggest a patch.)
In that case you should also update the comment above that `if let`, to say:
```rs
// Position and maximize the window.
```